tests/src/python/test_qgsrelation.py

```diff
@@ -184,9 +184,12 @@ def testValidRelationAfterChangingStyle(self):
         referencedLayer = relation.referencedLayer()
         valid = False
         for tab in referencedLayer.editFormConfig().tabs():
-            for t in tab.children():
-                if (t.type() == QgsAttributeEditorElement.AeTypeRelation):
-                    valid = t.relation().isValid()
+            if tab.type() == QgsAttributeEditorElement.AeTypeRelation:
```

Comment:
@pblottiere would you please have a look here? This test was failing on travis with:
```
Traceback (most recent call last):
  File "/root/QGIS/tests/src/python/test_qgsrelation.py", line 187, in testValidRelationAfterChangingStyle
    for t in tab.children():
AttributeError: 'QgsAttributeEditorRelation' object has no attribute 'children'
```
I cannot reproduce the issue locally, it looks like the `QgsAttributeEditorContainer` is not added and the `QgsAttributeEditorRelation` is added directly.

I patched the test here but maybe there is something else I can check.

Reply:
@elpaso I'm glad to see that you've fixed your issue :)

Reply:
@pblottiere I can't say I fixed it, I still cannot reproduce it locally so I had to bisect using a docker (that took hours!!!) and found the guilty commit, which contained an apparently harmless change (storing the style as QDomDocument instead of and XML string), I just reverted that commit. So far so good.